### DISCUSSION TITLE
2019 年总结 · 拥抱变化 · /2019-summary

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#585] [Announcements]
https://draveness.me/2019-summary

**Comment by nemoTyrant** [comment]
标注3、4、5反了哦

**Comment by Huweicai** [comment]
鼓掌鼓掌

**Comment by phpmaple** [comment]
希望2020 多出 Rust相关

**Comment by draveness** [comment]
> 希望2020 多出 Rust相关

哈哈哈哈，我尽量

**Comment by Devying** [comment]
一直关注楼主的文章，楼主工作2年多，就能写出一系列好文章，质量之高，内容之深，让我十分惊讶，可以想象楼主的对技术的学习方法，钻研精神，落实态度，知识锤炼都十分考究，让我这个工作6年多的coder自愧不如。向楼主学习🆙

**Comment by schwarzeni** [comment]
wow，才毕业两年就这么厉害了，会持续关注您的博客的，向您学习！

**Comment by draveness** [comment]
> 标注3、4、5反了哦

修复了，多谢

**Comment by draveness** [comment]
> 一直关注楼主的文章，楼主工作2年多，就能写出一系列好文章，质量之高，内容之深，让我十分惊讶，可以想象楼主的对技术的学习方法，钻研精神，落实态度，知识锤炼都十分考究，让我这个工作6年多的coder自愧不如。向楼主学习🆙

> wow，才毕业两年就这么厉害了，会持续关注您的博客的，向您学习！

感谢支持哈！

**Comment by beihai0xff** [comment]
借鉴大佬的学习路线努力了

**Comment by draveness** [comment]
> @wingsxdu 借鉴大佬的学习路线努力了

你这博客这么写不加 Reference 真的好么...

**Comment by stevenlee87** [comment]
加油加油！

**Comment by beihai0xff** [comment]
> @draveness
> > @wingsxdu 借鉴大佬的学习路线努力了
> 
> 你这博客这么写不加 Reference 真的好么...

看到邮件了，感谢批评指正。这篇文章灵感确实来自《Redis 快照复制使用子进程》，也参考了图片制作，最近也在看操作系统相关的书籍。在文章命名时我也想尽可能做出分类划分，很抱歉考虑不周，我会重新考虑一下博客写作风格。再次感谢批评。

**Comment by xiaoheiAh** [comment]
> @phpmaple
> 希望2020 多出 Rust相关

我也想说这个。

**Comment by draveness** [comment]
> > @phpmaple
> > 希望2020 多出 Rust相关
> 
> 我也想说这个。

👀

**Comment by Dream4ever** [comment]
虽然是一名用JS做前后端的全干工程师，读您的文章也能有不少收获，感谢分享，祝越来越好~

**Comment by pumpdude** [comment]
要常来大佬博客学习~

**Comment by lht1936** [comment]
请问博客的访问量用什么方法搜集和统计的。

**Comment by draveness** [comment]
> 请问博客的访问量用什么方法搜集和统计的。

GA

**Comment by draveness** [comment]
> 大佬 作图用的是什么软件

我真是想掏出 60 米长的大刀跟你聊聊🙂

**Comment by GavinCruise** [comment]
> @chazyu1996
> 大佬 作图用的是什么软件

作者已经写过了 https://draveness.me/sketch-and-sketch

**Comment by shellj** [comment]
相当优秀啊

**Comment by tanjunchen** [comment]
+1  鼓掌

**Comment by tanjunchen** [comment]
很多工程师包括作者都对其他领域的知识不感兴趣甚至认为除了技术的其他知识都不值得学习，但是随着接触的领域越来越多，发现了不同知识的价值，作者现在能以更开放的心态去主动学习更多技能。
认同！

**Comment by godliness** [comment]
你是我的榜样👍，向你学习！加油！

**Comment by draveness** [comment]
> 你是我的榜样👍，向你学习！加油！

感谢支持

**Comment by zionwu** [comment]
lz可以出篇文章讲下如何给Kubernetes贡献PR和成为member的过程吗？

**Comment by draveness** [comment]
> @zionwu
> lz可以出篇文章讲下如何给Kubernetes贡献PR和成为member的过程吗？

我尽量

**Comment by cfc4n** [comment]
太优秀了，像大神学习。

**Comment by ConteMan** [comment]
很喜欢的行文风格和技术态度。学习！

**Comment by sumaolin** [comment]
评论登陆好像有问题，在这篇文章登录跳转到首页，再回到这篇文章，还是未登录状态需要重新登陆，才正常跳转到这个页面

**Comment by draveness** [comment]
> @loveRhythm1990
> 博主的工作内容就是k8s吗？我也在做k8

不是

**Comment by draveness** [comment]
> @sumaolin
> 评论登陆好像有问题，在这篇文章登录跳转到首页，再回到这篇文章，还是未登录状态需要重新登陆，才正常跳转到这个页面

这个我也没有太好的办法...

**Comment by renjingneng** [comment]
十分感谢作者的为什么这样设计这个系列文章
非常好
希望2020年也能继续坚持下去
另外作者也可以开通捐款   让我们看了面费文章也能回馈作者

**Comment by draveness** [comment]
> 十分感谢作者的为什么这样设计这个系列文章
> 非常好
> 希望2020年也能继续坚持下去
> 另外作者也可以开通捐款 让我们看了面费文章也能回馈作者

感谢支持，网站上暂时就不放了，公众号里面有赞赏功能

**Comment by billychou** [comment]
最近工作上需要学习GO，使用google搜索问题的时候看到了作者的博客。博客内容质量很高，感谢作者。

**Comment by gaoshuai991** [comment]
想知道博主是怎么在工作之余有精力做这么多事情的。。。能分享一下吗

**Comment by draveness** [comment]
可以看下这个 https://draveness.me/few-words-time-management/

**Comment by NCZkevin** [comment]
太强了，看作者博客收货很多。想问大佬是在字节工作吗？

**Comment by feng99** [comment]
已关注大佬的公众号.

**Comment by Cherrison** [comment]
大佬你20年的年终总结呢？

**Comment by draveness** [comment]
> 大佬你20年的年终总结呢？

20 年没写总结，今年再写
